internal/policy/root.go

```diff
@@ -27,6 +27,49 @@ func InitializeRootMetadata(key *tuf.Key) *tuf.RootMetadata {
 	return rootMetadata
 }
 
+func AddRootKey(rootMetadata *tuf.RootMetadata, rootKey *tuf.Key) *tuf.RootMetadata {
+	rootMetadata.Keys[rootKey.KeyID] = rootKey
+	if _, ok := rootMetadata.Roles[RootRoleName]; !ok {
+		return InitializeRootMetadata(rootKey)
```

Comment:
hm, I don't know that we want to initialize the root role if it doesn't exist. Let's keep this separate for now, I think it's worth exploring this in the context of #45.

Reply:
Now AddRootKey just returns the given rootMetadata if the root role does not exist